modules/rhdh-plugins-reference/proc-setting-up-openshift-pipelines-to-download-or-view-sboms.adoc

```diff
@@ -0,0 +1,156 @@
+:_mod-docs-content-type: PROCEDURE
+[id="setting-up-openshift-pipelines-to-download-or-view-sboms_{context}"]
+
+= Setting up OpenShift Pipelines to view SBOM
+
+In Red Hat Developer Hub, under the CI tab, the Pipeline Runs section offers a feature to view Software Bill of Material (SBOM). This enhances transparency and control over your supply chain by listing all the software libraries a component utilizes. These libraries may provide specific functionality or facilitate development.
```

Comment:
Consider using the attribute (product for first occurence, product-short after):

```suggestion
In {product}, under the CI tab, the Pipeline Runs section offers a feature to view Software Bill of Material (SBOM). This enhances transparency and control over your supply chain by listing all the software libraries a component utilizes. These libraries may provide specific functionality or facilitate development.
```